library_generation/utilities.sh

```diff
@@ -216,3 +214,60 @@ download_fail() {
   >&2 echo "Fail to download ${artifact} from ${repo} repository. Please install ${artifact} first if you want to download a SNAPSHOT."
   exit 1
 }
+
+get_version_from_WORKSPACE() {
```

Comment:
Could you add unit tests for these utilities functions?

Reply:
added a couple for `BUILD` and `WORKSPACE` files